app/players/[id]/page.tsx

```diff
@@ -36,7 +36,7 @@ export default async function page({
   searchParams: URLSearchParams;
```

Comment:
`searchParams` is not of type `URLSearchParams`. See [next docs](https://nextjs.org/docs/app/api-reference/file-conventions/page#searchparams-optional)

Reply:
This one I will leave to you or Aki, this is used in 5 different places.

Reply:
@myssto for now I would like to leave as `URLSearchParams` and then changing better another time

Reply:
sure